android/app/build.gradle

```diff
@@ -65,6 +65,7 @@ dependencies {
     compile 'com.nimbusds:nimbus-jose-jwt:5.3'
     compile "com.google.code.gson:gson:2.8.2"
     compile 'com.microsoft.appcenter:appcenter:1.2.0'
```

Comment:
We don't need to add `om.microsoft.appcenter:appcenter` if we have `appcenter-crashes`